Classes/Emogrifier.php

```diff
@@ -533,12 +522,10 @@ private function removeImportantAnnotationFromNodeInlineStyle(\DOMElement $node)
      */
     private function getAllNodesWithStyleAttribute(\DOMXPath $xPath)
     {
-        $nodesWithStyleAttribute = $xPath->query('//*[@style]');
-        if ($nodesWithStyleAttribute === false) {
-            throw new \RuntimeException(
-                'Possibly malformed DOMXPath query expression or invalid DOMXPath object.',
-                1508961431
-            );
+        try {
+            $nodesWithStyleAttribute = $xPath->query('//*[@style]');
+        } catch (\RuntimeException $e) {
+            throw($e);
         }
```

Comment:
In this case, we can just not catch it and remove the catch and re-throw construct here (if I'm reading this correctly).